frontend/src/pages/pipelines/global/modelCustomization/ModelCustomizationForm.tsx

```diff
@@ -140,6 +144,54 @@ const ModelCustomizationForm: React.FC = () => {
     }),
   );
 
+  const [defaultStorageClass, defaultStorageClassLoaded, defaultStorageClassError] =
+    useDefaultStorageClass();
+  const isStorageClassesAvailable = useIsAreaAvailable(SupportedArea.STORAGE_CLASSES).status;
+  const preferredStorageClass = usePreferredStorageClass();
+  const openshiftDefaultStorageClass = useOpenshiftDefaultStorageClass(true);
```

Comment:
Any reason this can't be a simple "one stop shop" for storage classes -- this seems like a lot of granular checks for what should be a straight forward check, no?

Reply:
PVCs benefit from having their storage class auto selected if we dont provide a storage class. We can not provide undefined as an input to the pipeline so this is sort of ilab specific logic. I can move it into reusable hook, but this would only be used for ilab